reportssql/testing_report_functions.sql

```diff
@@ -1,5 +1,37 @@
 -- Testing Report
 
+DROP FUNCTION IF EXISTS Testing_Indicator1;
+
+DELIMITER $$
+CREATE FUNCTION Testing_Indicator1(
+    p_startDate DATE,
+    p_endDate DATE,
+    p_startAge INT(11),
+    p_endAge INT (11),
+    p_includeEndAge TINYINT(1),
+    p_gender VARCHAR(1),
+    p_hivResult VARCHAR(8),
+    p_testingEntryPoint VARCHAR(50)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE result INT(11) DEFAULT 0;
+
+    SELECT
+        COUNT(DISTINCT pat.patient_id) INTO result
+    FROM
+        patient pat
+    WHERE
+        patientGenderIs(pat.patient_id, p_gender) AND
+        getPatientHIVTestDateFromCounselingForm(pat.patient_id) > TIMESTAMPADD(MONTH, -3, CURDATE()) AND
```

Comment:
@tmvumbi2 The spec says: **If the patients last HIV test is not <= 3 months ago**. So this condition should be **getPatientHIVTestDateFromCounselingForm(pat.patient_id) < TIMESTAMPADD(MONTH, -3, CURDATE())** right?

See test cases 3 and 4:

https://docs.google.com/spreadsheets/d/1drmjK2JUgSeAOraH_jDIuE-kqMsNwN-lhZ1Q6QkWGGw/edit#gid=2038467389